client/gutenberg/extensions/publicize/editor.js

```diff
@@ -5,19 +5,50 @@
  *
  * Hooks into Gutenberg's PluginPrePublishPanel
  * to display Jetpack's Publicize UI in the pre-publish flow.
+ *
+ * It also hooks into our dedicated Jetpack plugin sidebar and
+ * displays the Publicize UI there.
+ */
+
+/**
+ * External dependencies
  */
+import { PanelBody } from '@wordpress/components';
+import { PluginPrePublishPanel } from '@wordpress/edit-post';
+import { PostTypeSupportCheck } from '@wordpress/editor';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
+import JetpackPluginSidebar from 'gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar';
 import PublicizePanel from './panel';
 import registerJetpackPlugin from 'gutenberg/extensions/presets/jetpack/utils/register-jetpack-plugin';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 
 export const name = 'publicize';
 
 export const settings = {
-	render: () => <PublicizePanel />,
+	render: () => (
+		<PostTypeSupportCheck supportKeys="publicize">
+			<JetpackPluginSidebar>
```

Comment:
It's cool how you get an overview of where publicize is being shown in a single place. The code becomes "feature" focused rather than "location" focused.